listener/pkg/event/http_test.go

```diff
@@ -29,12 +29,10 @@ import (
 func newTestListener(addr, component string, log logr.Logger,
 	verify listenerEvent.Verify,
 ) *listenerEvent.SKREventListener {
-	return &listenerEvent.SKREventListener{
-		Addr:          addr,
-		Logger:        log,
-		ComponentName: component,
-		VerifyFunc:    verify,
-	}
+	eventSource := make(chan event.GenericEvent)
```

Comment:
Why not move creating the channel to NewSKRListener() func? Then there is also no need for RegisterListenerComponent() func. We have less code then and it is more decoupled